frontend/src/views/WheresMyGene/components/Beta/style.ts

```diff
@@ -5,24 +5,28 @@ export const StyledNotification = styled(Notification)`
   ${(props: CommonThemeProps) => {
     const colors = getColors(props);
 
+    // beta intent does not exist for SDS Notification, but the colors do
     return `
-    .elevated {
-      border-color: ${colors?.beta[400]} !important;
-    }
+      #beta-notification {
```

Comment:
targetting specific id because using the generic `.elevated` class was messing with the copy link notification